### DISCUSSION TITLE
Support for OpenAI Structured Outputs

### DISCUSSION THREAD

**Discussion by guillaumemeyer** [langgenius/dify#7170] [Suggestion]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### 1. Is this request related to a challenge you're experiencing? Tell me about your story.

OpenAI recently released new options to their API to enforce structured outputs, namely forcing the LLM to return only JSON payloads with a specific format defined by a json-schema.
This option can also be applied to tools calling.
These options are especially useful in workflows scenarios, typically triggered from code.

### 2. Additional context or comments

https://openai.com/index/introducing-structured-outputs-in-the-api/

### 3. Can you help us with this feature?

- [ ] I am interested in contributing to this feature.

**Comment by magick93** [comment]
It looks like this has been added - https://github.com/langgenius/dify/pull/7258  - though I'm havent figured out how to use it yet.
